events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -0,0 +1,747 @@
+            var lightName = Entities.getEntityProperties(light, 'name').name;
+            if (lightName.indexOf(correctColor) === -1) {
+                Entities.callEntityServerMethod(light, "lightsOff");
+            }
+        });
+        correctHighlights.forEach(function(highlight) {
+            var highlightName = Entities.getEntityProperties(highlight, 'name').name;
+            if (highlightName.indexOf(correctColor) !== -1) {
+
+                Entities.callEntityServerMethod(highlight, "lightsOn");
+            }
+        });
+        var anyCorrect = isAnyAvatarCorrect(correctColor);
+        if (anyCorrect === 0) {
+            Script.setTimeout(function() {
+                prizeCalculator("everyone wrong");
+            }, FIRST_WAIT_TO_COUNT_AVATARS);
+        } else {
+            Script.setTimeout(function() {
+                Entities.callEntityServerMethod(bubble, "checkAnswer", [correctColor]);
+                correctCount = isAnyAvatarCorrect(correctColor);
+                updateAvatarCounter(true);
+            }, FIRST_WAIT_TO_COUNT_AVATARS);
+        }
+
+        Entities.callEntityServerMethod(answerText, "textUpdate", [formattedAnswer, true]);
+    }
+
+    function onWebEventReceived(event) {
+        if (typeof event === 'string') {
+            event = JSON.parse(event);
+            if (event.app === 'trivia') {
+                switch (event.type) {
+                    case 'begin':
+                        begin();
+                        break;
+                    case 'end':
+                        clearGame();
+                        break;
+                    case 'type':
+                        switch (event.value) {
+                            case "Any Type":
+                                type = null;
+                                break;
+                            case "Multiple Choice":
+                                type = "multiple";
+                                break;
+                            case "True or False":
+                                type = "boolean";
+                                break;
+                        }
+                        break;
+                    case 'difficulty':
+                        switch (event.value) {
+                            case "Any Difficulty":
+                                difficulty = null;
+                                break;
+                            case "Easy":
+                                difficulty = "easy";
+                                break;
+                            case "Medium":
+                                difficulty = "medium";
+                                break;
+                            case "Hard":
+                                difficulty = "hard";
+                                break;
+                        }
+                        break;
+                    case 'category':
+                        switch (event.value) {
```

Comment:
i'd prefer to see this in a JSON dictionary rather than a `switch`, but you can leave it this time around

Reply:
Another "Becky wrote this"...